grafana-stack/deploy.conf.default

```diff
@@ -0,0 +1,12 @@
+# Deployment configuration.
+#
+# It's interpreted as a shell script.
+# Sensible default value have been used, feel free to change them.
+
+REGISTRY=gcr.io/google-containers
```

Comment:
we should probably use a different registry, rather than the official one used for releases.